packages/integrations/deno/package.json

```diff
@@ -22,9 +22,7 @@
     "build": "astro-scripts build \"src/**/*.ts\" && tsc",
     "build:ci": "astro-scripts build \"src/**/*.ts\"",
     "dev": "astro-scripts dev \"src/**/*.ts\"",
-    "test:import": "deno test --allow-run --allow-env --allow-read --allow-net --ignore=test/dynamic-import.test.js ./test/",
-    "test:subprocess": "deno test --allow-run --allow-env --allow-net ./test/dynamic-import.test.js",
-    "test": "npm run test:import && npm run test:subprocess"
+    "test": "deno test --allow-run --allow-env --allow-read --allow-net ./test/"
```

Comment:
Nope, I actually updated this on purpose. I realized that we didn't need 2 separate scripts because the test that is testing dynamic imports launches its own subprocess that doesn't have the `--allow-read` flag, so it doesn't matter that the parent process that the test is running in does.